script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_10B.txt

```diff
@@ -1,6 +1,6 @@
 <Page1>
 A few hours have passed since the massive fire in front of the station.
-At the time of the fire, firefighters on the scene reported that it would take two days to extinguish the fire due to the intensity of the blaze, but the fire slowed down before midnight and was extinguished soon after.
+At the time of the fire, firefighters on the scene reported that it would take two days to extinguish it due to the intensity of the blaze, but it slowed down before midnight and was extinguished soon after.
```

Comment:
drop the first "it".
> it would take two days to extinguish due to the intensity

Reply:
At its peak, firefighters on scene reported it would take two days to extinguish due to the intensity of the blaze [...]